### PR TITLE
feat: add EnergyWebFoundation EWC network

### PR DESCRIPTION
**Pull request by jrhender**
This PR adds the Energy Web Foundations EWC network to the list of default networks.

Tested by running:
```
npm run start
```
Then
```
curl -X GET http://localhost:8081/1.0/identifiers/did:ethr:ewc:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736
```

### CODE REVIEW

**Comment by mirceanis** [comment]
Thanks for the submission!

Can you please add a couple of tests [here](https://github.com/uport-project/uport-did-driver/blob/master/src/__tests__/ethr.test.js) as well for
`did:ethr:ewc:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736`
and  `did:ethr:0xf6:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736` ?

Also, do you know which commit of ethr-did-registry you used for your registry at 0xE29672f34e92b56C9169f9D485fFc8b9A136BCE4 ?

**Comment by jrhender** [comment]
@mirceanis 
> Can you please add a couple of tests [here](https://github.com/uport-project/uport-did-driver/blob/master/src/__tests__/ethr.test.js) as well for
did:ethr:ewc:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736
and did:ethr:0xf6:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736 ?

Will do!

> Also, do you know which commit of ethr-did-registry you used for your registry at 0xE29672f34e92b56C9169f9D485fFc8b9A136BCE4 ?

I believe we used https://github.com/uport-project/ethr-did-registry/commit/c90638361a76d247d61ef4e3eb245a78cf587f91
Is this right @nichonien ?

**Comment by jrhender** [comment]
@mirceanis I've add the tests 👍

**Comment by nichonien** [comment]
> @mirceanis
> 
> > Can you please add a couple of tests [here](https://github.com/uport-project/uport-did-driver/blob/master/src/__tests__/ethr.test.js) as well for
> > did:ethr:ewc:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736
> > and did:ethr:0xf6:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736 ?
> 
> Will do!
> 
> > Also, do you know which commit of ethr-did-registry you used for your registry at 0xE29672f34e92b56C9169f9D485fFc8b9A136BCE4 ?
> 
> I believe we used [uport-project/ethr-did-registry@c906383](https://github.com/uport-project/ethr-did-registry/commit/c90638361a76d247d61ef4e3eb245a78cf587f91) Is this right @nichonien ?

@jrhender You are right, the deployed code could be found [here](https://explorer.energyweb.org/address/0xE29672f34e92b56C9169f9D485fFc8b9A136BCE4/contracts) just adding for quick reference.

File: src/__tests__/ethr.test.js [hunk 1]
```diff
@@ -247,6 +247,34 @@ describe('did:ethr driver', () => {
       const response = await request(app).get(`/1.0/identifiers/${did}`)
       expect(response.body.didDocument).toHaveProperty('verificationMethod')
     })
+
+
+    it('did:ethr:volta:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736', async () => {
+      expect.assertions(1)
+      const did = 'did:ethr:ewc:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736'
```
Comment (mirceanis) [inline src/__tests__/ethr.test.js:254]: I think there's a typo here

```suggestion
      const did = 'did:ethr:volta:0x3b0BC51Ab9De1e5B7B6E34E5b960285805C41736'
```

**Review by mirceanis — APPROVED** [review]
looks great!
Thank you for the contribution!

**Comment by mirceanis** [comment]
@jrhender I also made a PR to update the driver used by uniresolver.io to make use of your changes: https://github.com/decentralized-identity/universal-resolver/pull/286

**Comment by jrhender** [comment]
@mirceanis Great, thank you!

**Pull request merged**